CMSgov/mint-app
# [EASI-3253] - CSV filter groups

# EASI-3253

## Changes and Description

- Added until for csv export to filter data columns on the presence of a filter group selected
- Unit test

## Note
The UI rendering of filtered views will vary slightly as the rendering has some dynamic content on the presence of certain values of fields.  This seems unnecessary/out of scope/not worth the squeeze to replicate that in csv format

## How to test this change

Select a filter group on the csv export and verify that the data columns are filtered

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
